docs/dependent-apps.md

```diff
@@ -0,0 +1,289 @@
+# Using the Resin Supervisor for managing dependent applications
```

Comment:
I think "Using the Resin Supervisor to manage dependent applications" sounds better, not sure if it's more grammatically correct?